models/news.js

```diff
@@ -3,4 +3,21 @@ const db = require('../db/connection.js')
 exports.selectTopics = () => {
     return db.query(`SELECT * FROM topics;`)
     .then((topics) => topics.rows)
+}
+
+exports.selectArticles = () => {
+    return db.query(`
+    SELECT articles.author,
+    title,
+    articles.article_id,
+    topic,
+    articles.created_at,
+    articles.votes,
+    COUNT(*) AS comment_count FROM articles
+    JOIN comments
```

Comment:
You may need to look into an alternative JOIN to the default here - otherwise articles with no associated comments will be omitted